src/app/layout/header/header.component.html

```diff
@@ -79,8 +79,25 @@ <h4 class="mb-0 f-16 color-white">Tasks</h4>
                                             <mat-icon svgIcon="new-claim-icon"></mat-icon>
                                         </div>
                                         <div
-                                            class="media-body d-flex flex-row justify-content-between align-items-center">
```

Comment:
Why so big spaces? Maybe we should stick with one way, I mean `<div class="">...</div>` without additional enters/tabs/spaces. What do you think?